### PR TITLE
[POT] Added printing IE, MO and POT versions during POT launch

### PR DESCRIPTION
**Pull request by l-bat**

### CODE REVIEW

File: tools/pot/openvino/tools/pot/pipeline/pipeline.py [hunk 1]
```diff
@@ -31,6 +35,9 @@ def run(self, model):
         :param model: initial model
         :return model after applying all added algorithms
         """
+        logger.info(f'Inference Engine version:                {get_ie_version()}')
+        logger.info(f'Model Optimizer version:                 {get_mo_version()}')
+        logger.info(f'Post-Training Optimization Tool version: {get_pot_version()}')
```
Comment (l-bat) [inline tools/pot/openvino/tools/pot/pipeline/pipeline.py:40]: @AlexKoff88, please check that this is the right place to print versions

**Comment by AlexKoff88** [comment]
We have a version.py file in the root/opevino/tools/pot/ folder. I think we should remove it and adjust the setup.py logic accordingly and take the actual version in the same way as we do that in your PR. Can you please work on this as well?

**Review by AlexKoff88 — APPROVED** [review]

File: tools/pot/setup.py [hunk 2]
```diff
@@ -15,18 +14,23 @@
     long_description = fh.read()
 
 
-def read(*parts):
-    with codecs.open(os.path.join(here, *parts), 'r') as fp:
-        return fp.read()
+def generate_pot_version():
```
Comment (AlexKoff88) [inline tools/pot/setup.py:17]: I am not ok with code copy paste but if there is no way how to avoid it in such a case, let's merge it.

**Review by AlexKoff88 — APPROVED** [review]

**Comment by MaximProshin** [comment]
@azhogov , @AlexKoff88 , can it be merged?

**Comment by AlexKoff88** [comment]
> @azhogov , @AlexKoff88 , can it be merged?
I agree to merge this but the old CI issues prevent me from doing this. Hope, @azhogov can help.

**Review by ilya-lavrenov — APPROVED** [review]

**Pull request merged**